packages/common-utils/src/__tests__/__snapshots__/renderChartConfig.test.ts.snap

```diff
@@ -1,5 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`renderChartConfig containing CTE clauses should render a ChSql CTE configuration correctly 1`] = `"WITH TestCte AS (SELECT TimeUnix, Line FROM otel_logs) SELECT Line FROM TestCte"`;
```

Comment:
I added a nesting describe block, which changed the test title. The asserted value should be the same.